### ISSUE TITLE
[SVD] Is it possible to change the noise scheduler in StableVideoDiffusionPipeline?

### ISSUE THREAD

**Issue by JulianJuaner** [huggingface/diffusers#6535]
**What API design would you like to have changed or added to the library? Why?**
I tried to replace the SVD's scheduler with DDIM (default is EULER) but found that it could not output normal results. Is this normal? Because the original paper stated that the results were obtained using DDIM.

<img width="1028" alt="image" src="https://github.com/huggingface/diffusers/assets/37148692/f3e7f94f-2360-43c8-b5bf-8db8f82d2f3b">

**What use case would this enable or better enable? Can you give us a code example?**
My code here:
```python
model_ckpt = "stabilityai/stable-video-diffusion-img2vid-xt"
pipe = StableVideoDiffusionPipeline.from_pretrained(
    "stabilityai/stable-video-diffusion-img2vid-xt", torch_dtype=torch.float16, variant="fp16"
)
print(pipe.scheduler.compatibles) # it contains DDIMScheduler.
# If use default (EulerDiscreteScheduler), it works well
pipe.register_modules(scheduler=DDIMScheduler.from_config(pipe.scheduler.config))
pipe.enable_model_cpu_offload()

# Load the conditioning image
image = load_image("https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/svd/rocket.png?download=true")
image = image.resize((1024, 576))

device = "cuda"
pipe = pipe.to("cuda")
frames = pipe(image,
            num_frames=num_frames, decode_chunk_size=8,num_inference_steps=num_inference_steps,
            ).frames[0]
```

**Comment by JulianJuaner** [comment]
I find why. SVD large-scale training is only capable of continuous noise schedulers.

**Comment by patrickvonplaten** [comment]
Also see: https://github.com/huggingface/diffusers/discussions/6541

**Comment by Mowenyii** [comment]
I have the same question, is there any way to change to DDIM scheduler?

**Comment by lim142857** [comment]
So it is not possible to perform DDIM inversion using SVD?

**Comment by jufeif** [comment]
> I have the same question, is there any way to change to DDIM scheduler?

Have you solved it?

**Comment by ustcuna** [comment]
I was trying to change to DPMSolverMultistepScheduler, but found that it would dump with IndexError.
![image](https://github.com/huggingface/diffusers/assets/89436425/dd100239-5f50-4171-88e4-9258389678f4)

I tried to print(pipe.scheduler.compatibles), I saw <class 'diffusers.schedulers.scheduling_dpmsolver_multistep.DPMSolverMultistepScheduler'>.
So, was this error due to SVD only capable of continuous noise schedulers, but not discrete-time schedulers? Or is there any configs we need to set to enable DPMSolverMultistepScheduler for SVD?

**Comment by yiyixuxu** [comment]
we have edm_dpm_multistep now, do you want to give it a try? https://github.com/huggingface/diffusers/blob/main/src/diffusers/schedulers/scheduling_edm_dpmsolver_multistep.py

**Comment by ustcuna** [comment]
> we have edm_dpm_multistep now, do you want to give it a try? https://github.com/huggingface/diffusers/blob/main/src/diffusers/schedulers/scheduling_edm_dpmsolver_multistep.py

Hi @yiyixuxu , thank you so much for the quick response. I've tried with EDMDPMSolverMultistepScheduler, this time we can run successfully but the output seems not as good as the default scheduler, almost the same quality just like DDIM here. Anyway, thx again!

**Comment by yiyixuxu** [comment]
Were you able to get the same result with edm_euler? https://github.com/huggingface/diffusers/blob/main/src/diffusers/schedulers/scheduling_edm_euler.py

**Comment by ustcuna** [comment]
> Were you able to get the same result with edm_euler? https://github.com/huggingface/diffusers/blob/main/src/diffusers/schedulers/scheduling_edm_euler.py
Thx for the advice. I just tried w/ EDMEulerScheduler, the video quality seems better than DDIM and EDMDPMSolverMultistepScheduler, but still not as good as default scheduler.

**Comment by yiyixuxu** [comment]
@ustcuna can you share your script? with EDMEulerScheduler I think you should get be getting same result as the default scheduler
I haven't really tried it out so would like to look into this

**Comment by ustcuna** [comment]
> @ustcuna can you share your script? with EDMEulerScheduler I think you should get be getting same result as the default scheduler I haven't really tried it out so would like to look into this

Hi @yiyixuxu ,sure, thx for your help! Here is a very simple version of my script. When changing to EDMEulerScheduler, I tried to use all default configs once, and another time I set the args "prediction_type" to "v_prediction" while other args for EDMEulerScheduler remains as default.
```
    import torch

    from diffusers import StableVideoDiffusionPipeline, EDMEulerScheduler
    from diffusers.utils import load_image, export_to_video

    model_id = "/home/models/stable-video-diffusion-img2vid-xt-1-1/"
    pipe = StableVideoDiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float32, variant="fp32")
    c = pipe.scheduler.config
    # c["prediction_type"] = "v_prediction"
    pipe.scheduler = EDMEulerScheduler.from_config(c)
    print("changed scheduler to " + pipe.scheduler.__class__.__name__)
    
    # the rocket image
    image = load_image("./test_image.png")
    image = image.resize((1024, 576))
    
    generator = torch.Generator(device).manual_seed(4)
    frames = pipe(image, num_frames=25, num_inference_steps=30, decode_chunk_size=8, generator=generator).frames[0]

    export_to_video(frames, "generated_"+pipe.scheduler.__class__.__name__+".mp4", fps=10)
```

**Comment by zideliu** [comment]
> I have the same question, is there any way to change to DDIM scheduler?

I also want to know it.
